aws/cluster/data.tf

```diff
@@ -20,7 +20,7 @@ data "aws_security_group" "masters" {
   }
 }
 
-data "aws_ebs_volume" "etcd-volumes" {
+data "aws_ebs_volume" "etcd-volumes" { // Wandera
```

Comment:
I don't know if those comments make sense as our fork of karch is really fundamentally different than upstream in terms of network layer handling.

Reply:
The comments are for the review, so we can see what is specific to us and if we want to change something. To be removed when all agreed